plugins/c8y_log_plugin/src/main.rs

```diff
@@ -238,23 +213,26 @@ fn create_init_logs_directories_and_files(
         "tedge",
         "tedge",
         0o755,
+        None,
     )?;
     // creating c8y directory
-    create_directory_with_user_group(&format!("{config_dir}/c8y"), "tedge", "tedge", 0o755)?;
-    // creating c8y-log-plugin.toml
+    create_directory_with_user_group(&format!("{config_dir}/c8y"), "root", "root", 0o755)?;
```

Comment:
According to the Didier's comment https://github.com/thin-edge/thin-edge.io/pull/1217#issuecomment-1165430950.
```suggestion
    create_directory_with_user_group(&format!("{config_dir}/c8y"), "root", "root", 0o1777)?;
```

Reply:
fixed